docs/_data/nav_docs.yml

```diff
@@ -54,6 +54,8 @@
       title: Integrating with Other Libraries
     - id: accessibility
       title: Accessibility
+    - id: cross-origin-errors
```

Comment:
Does it need to be in the table of contents? Normally we don't add items for warnings. Those who have them will read this by following a link, those who don't probably don't need to know this. Every item in TOC adds some extra mental overhead for people learning React.